tests/conftest.py

```diff
@@ -68,18 +68,21 @@ class Capture(object):
     def __init__(self, capfd):
         self.capfd = capfd
         self.out = ""
+        self.err = ""
 
-    def _flush_stdout(self):
+    def _flush(self):
         sys.stdout.flush()
```

Comment:
The extra effort was mostly to work around issues on Windows. After this I plan to submit a PR which replaces `std::cout` with `py::print` everywhere, which fixes all output issues on Windows. At that point this entire `_flush()` method will be deleted since pytest's `capfd` already takes care of it.